src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -325,20 +332,9 @@ define('pako-global', ['pako'], function(pako) {
     diagramViewerDeferred.reject();
   });
 
-  // Return a diagram in XML format if is present in the URL.
-  function getDiagramXMLFromURL() {
-    if (window.location.hash.substring(0, 2) === '#R') {
-        // We need to exclude first 2 letters from hash (#R).
-        let decodedData = decodeURIComponent(window.location.hash.substring(2));
-        let graph = new Graph();
-        let rawDiagram = graph.decompress(decodedData);
-        return rawDiagram;
-    }
-    return null;
-  };
-
-  function getDiagramXML(container) {
-    return getDiagramXMLFromURL() || container.data('model');
+  function getDiagramXML(url, container) {
```

Comment:
You're calling this function with ``window.location.hash`` above so the first parameter seems to be a hash not a full URL. Which part is right?